project2-beat-that/script.js

```diff
@@ -1,4 +1,65 @@
+//for main function
+var player_score1 = 0;
+var player_score2 = 0;
+var playerFinalScore = 0;
+
+//players current score
+var player1_score = 0;
+var player2_score = 0;
+
+// players total score
+var player1_total = 0;
+var player2_total = 0;
+
+// counters
+var counter = 0;
+var remainder = 0;
+var player2 = 0;
+
+// arrray for score recording
+var allPlayersTotal = [0, 0];
+
+var rollDice = function () {
+  //player 1 first roll
+  var randomRollDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomRollDecimal) + 1;
+  return randomInteger;
+};
```

Comment:
we showed this function line by line to make it clearer what is happening, but once you understand it, i think its succinct enough that the body can be reduced to one line

`return Math.floor(Math.random() * 6) + 1;`